chatmm/src/main/java/com/likeminds/chatmm/media/customviews/CanvasView.kt

```diff
@@ -272,19 +271,19 @@ internal class CanvasView : View, OnRotationGestureListener {
         val textX = this.textX
         val textY = this.textY
 
-        val paintForMeasureText = Paint()
+//        val paintForMeasureText = Paint()
 
         // Line break automatically
-        val textLength = paintForMeasureText.measureText(text)
-        val lengthOfChar = textLength / text.length.toFloat()
-        val restWidth = canvas.width - textX // text-align : right
-        val numChars =
-            if (lengthOfChar <= 0) 1 else floor((restWidth / lengthOfChar).toDouble())
-                .toInt() // The number of characters at 1 line
-        val modNumChars = if (numChars < 1) 1 else numChars
-        var y = textY
-        var i = 0
-        val len = text.length
+//        val textLength = paintForMeasureText.measureText(text)
+//        val lengthOfChar = textLength / text.length.toFloat()
+//        val restWidth = canvas.width - textX // text-align : right
+//        val numChars =
+//            if (lengthOfChar <= 0) 1 else floor((restWidth / lengthOfChar).toDouble())
+//                .toInt() // The number of characters at 1 line
+//        val modNumChars = if (numChars < 1) 1 else numChars
+//        var y = textY
+//        var i = 0
+//        val len = text.length
```

Comment:
why commented?

Reply:
as this code is not being used. removing the code from the code base